src/routes/users.js

```diff
@@ -1,54 +1,64 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
-const { check, validationResult } = require('express-validator');
-const User = require('../models/User');
+const { check, validationResult } = require("express-validator");
+const User = require("../models/User");
 
 /**
  * @route POST api/users
  * @access Public
  * @desc Register users
- * @todo TODO: Hash password
  */
-router.post('/', [
-  check('name', 'Name is required').not().isEmpty(),
-  check('email', 'Please add a valid email').isEmail(),
-  check('password', 'Please add a password with 6 or more characters').isLength({
-    min: 6
-  }),
-], async (req, res) => {
-
-  // express-validator
-  const errors = validationResult(req);
-  if (!errors.isEmpty()) {
-    return res.status(400).json({errors: errors.array()});
-  }
-  
-  const { name, email, password } = req.body;
-  
-  try {
-    // See if user exists
-    let user = await User.findOne({ email });
-
-    if (user) {
-      return res.status(400).json({errors: [{ msg: `User with ${email} email already exists!` }]});
```

Comment:
The prettier did it's work so that is why it shows as deleted.  I will comment on what I changed.